packages/ReadWrite/NodeFinder/NodeUsageFinder.php

```diff
@@ -25,7 +25,7 @@ public function __construct(
 
     /**
      * @param Node[] $nodes
-     * @return Variable[]
+     * @return never[]|Node[]
```

Comment:
@TomasVotruba handle UnionType https://github.com/rectorphp/rector-src/pull/393/commits/273dc5d09243b56449a889c163c775c602487ce5#diff-e21a1c8f1784f36751a42d937959b979ca4fed5ddf8f05ad74c8c4e72dc84dc3R55 seems make addition of `never[]` on various places.

Reply:
handled at https://github.com/rectorphp/rector-src/pull/393/commits/b17ef4f966bb5228ef3f2b74025c284bc3681cf3